code/modules/deathmatch/deathmatch_loadouts.dm

```diff
@@ -12,6 +12,9 @@
 
 /datum/outfit/deathmatch_loadout/pre_equip(mob/living/carbon/human/user, visualsOnly = FALSE)
 	. = ..()
+#ifdef UNIT_TESTS
+	return
+#endif
```

Comment:
I'll take a swing at this yeah

Reply:
so I removed the check here and am instead simply excluding all DM loadouts. manually specifying exclusions would be a less optimal approach since part of my goal was to make it easier for downstream servers (i.e. mine) to create deathmatch loadouts that change species.

Reply:
works fine